x-pack/plugins/apm/public/components/shared/apm_header_action_menu/labs/labs_flyout.tsx

```diff
@@ -54,7 +54,7 @@ export function LabsFlyout({ onClose }: Props) {
         return settingsEditableConfig[key].requiresPageReload;
       });
 
-      await saveAll();
+      await saveAll({ trackMetricName: 'labs_save' });
```

Comment:
Would it be useful to know which features are enabled/disabled when user clicks on `Save`?

Reply:
@yngrdyn this is already handled by the advanced settings itself. Check https://github.com/elastic/telemetry/pull/1612